js/src/Arrow.externs.js

```diff
@@ -79,6 +79,12 @@ Value.prototype.gteq;
 Value.prototype.lteq;
```

Comment:
Is there any prior art for naming these methods in JavaScript? (in Python we have lt, le, gt, ge, eq, ne)

Reply:
Not that I'm aware of. We (CCRi) chose them arbitrarily when first implementing this. I wouldn't be opposed to renaming to those two-letter names to be more consistent and concise.

Reply:
@trxcllnt do you have an opinion?

Reply:
@wesm I'd be partial to the two-letter names if it means we're more aligned with other data-science tools. But I'm also not the primary consumer of these APIs yet (we're focused on GPU dataframe ops), so I feel my vote should count less than @TheNeuralBit's here.

Reply:
ok I'll go ahead and update the names here. Thanks for the suggestion Wes, it's good to make these changes now.